pyexchange/tethfinex.py

```diff
@@ -207,7 +209,7 @@ def get_config(self):
     def get_orders(self, pair: str) -> List[Order]:
         assert(isinstance(pair, str))
 
-        result = self._get_orders(f"/trustless/v1/r/orders/t{pair}")
+        result = self._get_orders(f"/trustless/v1/r/orders/:{pair}")
```

Comment:
it seems like Trustless Ethfinex now expects pairs as ETHDAI instead tETHDAI, can you please confirm that?

Reply:
The API expects tETHDAI still

Reply:
Got it, so the t{pair} format should remain

Reply:
I fix it, i edited it because i saw `:` in ethfinex docs but i have just checked into `exf-api-node` code and there is `t`.